src/services/organizeImports.ts

```diff
@@ -135,6 +143,14 @@ namespace ts.OrganizeImports {
         }
     }
 
+    function hasModuleDeclarationMatchingSpecifier(sourceFile: SourceFile, moduleSpecifier: Expression) {
+        const moduleSpecifierText = isStringLiteral(moduleSpecifier) && moduleSpecifier.text;
+        return isString(moduleSpecifierText) && some(sourceFile.statements, statement =>

```

Comment:
Do you need to handle the new patternAmbientModule as well apart from the m,odule specifier text

Reply:
Hmm, I don't _think_ so, but I'm not sure exactly what rule says why. Can you come up with an example where that would be necessary? E.g., this doesn't work:

```ts
import 'typescript';
declare module 'type*' {
  interface Node {
    mySpecialNodeAugmentation: string;
  }
}
```